spec/spec_helper.rb

```diff
@@ -0,0 +1,14 @@
+# require "test/unit"
+require "cbx_loco"
+require "rspec"
+require "byebug"
```

Comment:
Unneeded require, as byebug is not used in the code. I understand that it was used during development, but the require should not make it to "production" code.